docs/contributing/contributing-code/contributing-code-schema-changes/README.md

```diff
@@ -51,28 +51,30 @@ Visit the [docs](https://github.com/radius-project/docs/) and [samples](https://
 
 # Testing schema changes locally
 
-If you would like to test that your schema changes are compilable in a Bicep template, you can do so by publishing them to an OCI registry using the [Bicep CLI](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/). 
+If you would like to test that your schema changes are compilable in a Bicep template, you can do so by publishing them to a file system using the [Bicep CLI](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/). 
```

Comment:
Same as above. Updating the instructions to use a file system instead of an OCI compliant registry